inlang/source-code/importers/icu2-message-format/src/plugin.ts

```diff
@@ -0,0 +1,149 @@
+import * as MF from "messageformat"
+import { displayName, description } from "../marketplace-manifest.json"
+import { PluginSettings } from "./settings.js"
+import { AST } from "@inlang/messages"
```

Comment:
@martin-lysk what is this AST object? I can't find the `@inlang/messages` package